README.rst

```diff
@@ -436,6 +436,12 @@ Using the ``cardano-node`` Haskell packages
 If you want to use the ``cardano-node`` Haskell packages from another project, you should use `CHaP <https://github.com/input-output-hk/cardano-haskell-packages>`_ to get the packages defined in this repository.
 Please note that you may need to use any ``source-repository-package`` stanzas defined in ``cabal.project``, although we will endeavour to keep these to an absolute minimum.
 
+****
+Style guide
+****
+
+The `style guide <https://github.com/input-output-hk/cardano-node/wiki/Style-guide>` for can be found in the wiki.
+
```

Comment:
I would like that `plutus`, `ouroboros-network`, the future `ouroboros-consensus` and `cardano-ledger` are consistent in where style guide can be found.

* `plutus`: `STYLEGUIDE.docs`
* `ouroboros-network`: `docs/StyleGuide.md`
* `ouroboros-consensus` (inside `ouroboros-network` repo): `ouroboros-consensus/docs/StyleGuide.md`
* `cardano-node`: a wiki.

I would rather like to move the wiki page to `STYLEGUIDE.md` or at least put the link in the `STYLEGUIDE.md` document.

Reply:
Would it suffice to rename `STYLE.md` to `STYLEGUIDE.md`?

Reply:
Yes, I'd use `/STYLEGUIDE.md`; I plan do the same once `ouroboros-consensus*` migrates to its own repo.

Reply:
Renamed.